src/types.ts

```diff
@@ -0,0 +1,7 @@
+import * as express from 'express'
+import { Contact } from './models'
+
+export interface Req extends express.Request {
```

Comment:
did something like this on #287 
https://github.com/stakwork/sphinx-relay/pull/287/files#diff-3e0f247bfd5389158886d6f0bff49895160ba71a8dc3d874a5760f833019e568 (didnt know the type when making this lol)

Reply:
note: diff can take a while to load and scroll down

Reply:
nice, ya the `req.owner` is added in by the auth middleware, and is used in pretty much every route, so wanted that to be strongly typed